task/fbc-validation/0.1/README.md

```diff
@@ -0,0 +1,27 @@
+# fbc-validation task
+
+## Description:
+The fbc-validation task will ensure FBC (File based catalog) components uniquely linted to ensure they're properly
+constructed as part of the build pipeline. To validate the image in build pipeline, Skopeo is used to extract
+information and then contents are checked using the OpenShift Operator Framework.
```

Comment:
```suggestion
information and then contents are checked using the OpenShift Operator Framework opm CLI tool.
```

Reply:
Shouldn't it be?
`...Framework's opm CLI tool.`
to denote that the opm tool is provided by OpenShift Operator Framework.

Reply:
Yeah,  you're right that makes more sense